integ-test/src/test/java/org/opensearch/sql/legacy/PrettyFormatResponseIT.java

```diff
@@ -195,24 +213,31 @@ private Set<String> fields(String... fieldNames) {
 
   @Test
   public void selectNestedFields() throws IOException {
-    JSONObject response = executeQuery(
-        String.format(Locale.ROOT, "SELECT nested(message.info), someField FROM %s",
-            TestsConstants.TEST_INDEX_NESTED_TYPE));
+    JSONObject response =
+        executeQuery(
+            String.format(
+                Locale.ROOT,
+                "SELECT nested(message.info), someField FROM %s",
+                TestsConstants.TEST_INDEX_NESTED_TYPE));
 
     List<String> fields = Arrays.asList("nested(message.info)", "someField");
     assertContainsColumns(getSchema(response), fields);
     assertContainsData(getDataRows(response), fields);
 
-    // The nested test index being used contains 5 entries but one of them has an array of 2 message objects, so
+    // The nested test index being used contains 5 entries but one of them has an array of 2 message
+    // objects, so
     // we check to see if the amount of data rows is 6 since that is the result after flattening
```

Comment:
This should probably be changed to a javadoc because this has been split into 3 awkward lines

Reply:
Javadoc is generally reserved for before a class/function to define the use/functionality. I think I'll leave it as is. It's not a good change by spotless, but doesn't hurt things.